mpmt_server.c

```diff
@@ -189,6 +189,7 @@ static void exit_normaly()
 #ifdef MULTICHILD
     child_data = NULL;
     dettach_childs_queue(childs_queue);
+    ci_acl_destroy();
```

Comment:
The ci_acl_destroy function is declared in acl.h file. This include is not needed for mpmt_server.c.
Please double check that including this file is not required to avoid warnings while the mpmt_server.c file is compiled.
If required please add it in include files list.